.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -248,9 +275,9 @@ jobs:
           curl -S -s http://${{ env.APP_ENDPOINT }}/client-call/
 
       - name: Build Gradlew
-        run: |
-          source ${{ env.TEST_RESOURCES_FOLDER }}/.github/workflows/util/execute_and_retry.sh
-          execute_and_retry 2 "./gradlew"
+        uses: ./.github/workflows/actions/execute_and_retry
+        with:
+          command: "./gradlew"
```

Comment:
nit: Can use more retries here for safety

Reply:
Done